Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1029,6 +1038,25 @@ namespace Babylon
         return Napi::External<TextureData>::New(info.Env(), new TextureData());
     }
 
+    Napi::Value NativeEngine::CreateDepthTexture(const Napi::CallbackInfo& info)
+    {
+        const auto texture = info[0].As<Napi::External<TextureData>>().Data();
+        uint16_t width = static_cast<uint16_t>(info[1].As<Napi::Number>().Uint32Value());
+        uint16_t height = static_cast<uint16_t>(info[2].As<Napi::Number>().Uint32Value());
+        bgfx::FrameBufferHandle frameBufferHandle{};
+
+        // This is WIP
```

Comment:
Is there an issue to track?

Reply:
Yes https://github.com/BabylonJS/BabylonNative/issues/398